content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -357,5 +523,48 @@ To make changes to the instance level bucket storage for your W&B Server:
 1. Enter the new **Bucket Name**.
 1. Optionally, enter the **Path** to use in the new bucket.
 1. Click **Save**
+
+{{% /tab %}}
+{{% tab header="Environment variable" %}}
+
+To configure instance level BYOB, set the `GORILLA_SUPPORTED_FILE_STORES` environment variable to the bucket location, then restart W&B. 
 {{% /tab %}}
 {{< /tabpane >}}
+
+{{% alert %}}
+W&B recommends that you use a Terraform module managed by W&B to provision a storage bucket along with the necessary access mechanism and related IAM permissions:
+
+* [AWS](https://github.com/wandb/terraform-aws-wandb/tree/main/modules/secure_storage_connector)
+* [GCP](https://github.com/wandb/terraform-google-wandb/tree/main/modules/secure_storage_connector)
+* Azure - [Instance level BYOB](https://github.com/wandb/terraform-azurerm-wandb/tree/main/examples/byob) or [Team level BYOB](https://github.com/wandb/terraform-azurerm-wandb/tree/main/modules/secure_storage_connector)
+{{% /alert %}}
+
+#### Team level BYOB
+After you [determine the storage location](#determine-the-storage-location) for your bucket, you can use the W&B App to configure team level BYOB while creating the team.
+
+{{% alert %}}
+- After a team is created, its storage cannot be updated.
+- For Instance level BYOB, refer to [Instance level BYOB]({{< relref "#instance-level-byob" >}}) instead.
+{{% /alert %}}
+
+1. <!--If you are configuring CoreWeave storage for Multi-tenant Cloud, switch to the browser window where you previously began to create the new team to find the W&B organization ID previously. Otherwise, l-->Log in to W&B as a user with the `admin` role, click the icon at the top left to open the left navigation, then click **Create a team to collaborate**.
+1. Provide a name for the team.
+1. Set **Storage Type** to **External storage**.
+1. Click **Bucket location**. Select an existing bucket or click **Add bucket** at the bottom to create a new bucket.
+    To add a new bucket:
```

Comment:
Aww. Do we now have lettered ordered lists? Might make it more clear?

<img width="859" alt="Screenshot 2025-06-27 at 1 29 33 PM" src="https://github.com/user-attachments/assets/7e7d59d5-69d5-4e8d-8fa7-705320052784" />